hub-config/target-data.json

```
@@ -0,0 +1,9 @@
+{
+  "schema_version": "https://raw.githubusercontent.com/hubverse-org/schemas/main/v6.0.0/target-data-schema.json",
+  "observable_unit": [
+    "target",
+    "time_idx",
+    "location"
+  ],
+  "date_col": "time_idx"
+}
```

```suggestion
}

```
New line at the end of the file is missing